### ISSUE TITLE
Recurring Events - Pricing Settings Not Available to Update

### ISSUE THREAD

**Issue by bleven** [tendenci/tendenci#482]
On recurring events, there is no way to make a global change to a price setting once a recurring event is set up. 

All the other tabs in in Events (Location, Speakers, etc.) have a radio button selection for 
"Apply changes to:
- This event only
- This and the following events in series
- All events in series"

The pricing tab does not, leaving no way to make a Price Change to all events in a series. 

In summary - Requesting to have the "Apply Changes" radio buttons added to the Pricing Tab.

**Comment by jennyq** [comment]
Resolved https://github.com/tendenci/tendenci/commit/3d96203e8670dda75899b41dc2a2a4b02ead61ef
